Xamarin.Essentials/HapticFeedback/HapticFeedback.shared.cs

```diff
@@ -5,10 +5,45 @@ namespace Xamarin.Essentials
     public static partial class HapticFeedback
     {
         public static void Perform(HapticFeedbackType type = HapticFeedbackType.Click)
+            => PlatformPerform(type);
+
+        public static HapticFeedbackGenerator PrepareGenerator(HapticFeedbackType type = HapticFeedbackType.Click)
+#if MONOANDROID || WINDOWS_UWP || __MACOS__ || TIZEN
```

Comment:
If possible, can we move this to platform specific files? This will keep the shared code files clean and also forms a "interface" of sorts. You can create a a HapticFeedback.android.uwp.macos.tizen.cs just or these to reduce code and files.

Reply:
Yes, well, I'll split this into specific platforms files